src/controllers/bmdashboard/bmReusableController.js

```diff
@@ -47,17 +47,10 @@ const bmReusableController = function (BuildingReusable) {
         }
     };
 
-    const purchaseReusable = async (req, res) => {
```

Comment:
Hey, I am not able to build the app. looks like you deleted purchaseReusable but its still being return from the controller which is causing an error.

![PR 773](https://github.com/OneCommunityGlobal/HGNRest/assets/87269725/61d8c2c3-7d57-4925-84b1-be2afa43e057)